src/test/java/org/gridsuite/modification/server/modifications/TwoWindingsTransformerCreationTest.java

```diff
@@ -0,0 +1,265 @@
+                                        .x(0)
+                                        .g(0)
+                                        .b(0)
+                                        .alpha(0.)
+                                        .build()
+                        )).build())
+                .ratioTapChanger(RatioTapChangerCreationInfos.builder()
+                        .lowTapPosition(5)
+                        .tapPosition(6)
+                        .regulating(true)
+                        .targetDeadband(1.)
+                        .regulatingTerminalId("v2load")
+                        .regulatingTerminalVlId("v2")
+                        .regulatingTerminalType("LOAD")
+                        .loadTapChangingCapabilities(true)
+                        .targetV(5.)
+                        .steps(List.of(TapChangerStepCreationInfos.builder()
+                                        .index(5)
+                                        .rho(1)
+                                        .r(0)
+                                        .x(0)
+                                        .g(0)
+                                        .b(0)
+                                        .build(),
+                                TapChangerStepCreationInfos.builder()
+                                        .index(6)
+                                        .rho(1)
+                                        .r(0)
+                                        .x(0)
+                                        .g(0)
+                                        .b(0)
+                                        .build(),
+                                TapChangerStepCreationInfos.builder()
+                                        .index(7)
+                                        .rho(1)
+                                        .r(0)
+                                        .x(0)
+                                        .g(0)
+                                        .b(0)
+                                        .build(),
+                                TapChangerStepCreationInfos.builder()
+                                        .index(8)
+                                        .rho(1)
+                                        .r(0)
+                                        .x(0)
+                                        .g(0)
+                                        .b(0)
+                                        .build()
+                        ))
+                        .build())
+                .build();
+    }
+
+    @Override
+    protected MatcherTwoWindingsTransformerCreationInfos createMatcher(ModificationInfos modificationInfos) {
+        return MatcherTwoWindingsTransformerCreationInfos.createMatcherTwoWindingsTransformerCreationInfos((TwoWindingsTransformerCreationInfos) modificationInfos);
+    }
+
+    @Override
+    protected void assertNetworkAfterCreation() {
+        assertNotNull(getNetwork().getTwoWindingsTransformer("new2wt"));
```

Comment:
modification-server traces: "2 voltage level resources updated in 50 ms"
We can assert theses changes here:
```
assertEquals(1, getNetwork().getVoltageLevel("v1").getTwoWindingsTransformerStream().filter(transformer -> transformer.getId().equals("new2wt")).cou
assertEquals(1, getNetwork().getVoltageLevel("v2").getTwoWindingsTransformerStream().filter(transformer -> transformer.getId().equals("new2wt")).count());
```